src/types/Phone.ts

```diff
@@ -0,0 +1,41 @@
+export interface Phone {
+  id: string;
+  category: string;
+  namespaceId: string;
+  name: string;
+  capacityAvailable: string[];
+  capacity: string;
+  priceRegular: number;
+  priceDiscount?: number;
+  colorsAvailable: string[];
+  color: string;
+  images: string[];
+  description: Array<{
+    title: string;
+    text: string[];
+  }>;
+  screen: string;
+  resolution: string;
+  processor: string;
+  ram: string;
+  camera: string;
+  zoom: string;
+  cell: string[];
+}
+
+
+export interface PhoneState {
+  phones: Phone[];
+  loading: boolean;
+  error: string | null;
+}
+
+export const initialState: PhoneState = {
+  phones: [],
+  loading: false,
+  error: null,
+};
+
+export interface RootState {
+  phones: PhoneState;
+}
```

Comment:
I see several problems with this file. Let's start with the fact that in the file with the Phone type we have a constant and the root state for some reason.
In fact, the only thing that should be left here is `Phone`. 
`PhoneState` and `initialState` can be in the slice, they should be used only there. 
`RootState` will most likely be located where we create this store and look like this
`export type RootState = ReturnType<typeof rootReducer>;`